### DISCUSSION TITLE
ingress dns ubuntu ?

### DISCUSSION THREAD

**Discussion by jetersen** [rancher-sandbox/rancher-desktop#1953] [Q&A]
Having trouble finding docs for how to get DNS lookup work for ingress hosts with Rancher Desktop and Lima on ubuntu.

**Comment by adamkpickering** [comment]
I'm not sure what you're asking. What are you trying to do?

**Reply by jetersen** [reply]
See
https://github.com/rancher-sandbox/rancher-desktop/issues/1668
https://github.com/rancher-sandbox/rancher-desktop/issues/576

The issue is that port 80 and 443 require permission on the host for ingress to work.
